INSTALL.md

```diff
@@ -242,6 +242,12 @@ $ cabal repl
 Ok, modules loaded: Main.
 *Main>
 ```
+NOTE : If you get the following error:
```

Comment:
Why "If"? Doesn't it come all the time, because we are using FFI? We can directly recommend `cabal repl --ghc-options "-fobject-code"` in the above example.

Reply:
It didn't come when I followed the instructions on my local machine.